play/src/components/Timer.tsx

```diff
@@ -63,19 +70,26 @@ export default function Timer({
     }
   }
 
-  // generates timer string (needs to ensure that seconds are always 2 digits and don't show as 60)
-  function getTimerString(currentTimeInput: number) {
-    let sec = 0;
-    let secStr = '00';
-    let min = 0;
-    if (currentTimeInput >= 0) {
-      min = Math.floor(currentTimeInput / 60);
-      sec = Math.ceil(currentTimeInput % 60);
-      if (sec === 60) sec = 0;
-      secStr = sec < 10 ? `0${sec}` : `${sec}`;
+  const timerString = useMemo(() => {  
```

Comment:
I messed up the implementation of `useMemo` originally here so I've pushed a commit that resolves it. Sorry for any confusion caused by that.